shell/common/shell_unittests.cc

```diff
@@ -1430,7 +1379,7 @@ TEST_F(ShellTest, WaitForFirstFrameTimeout) {
 
   RunEngine(shell.get(), std::move(configuration));
   fml::Status result =
-      shell->WaitForFirstFrame(fml::TimeDelta::FromMilliseconds(10));
+      shell->WaitForFirstFrame(fml::TimeDelta::Zero());
```

Comment:
is the assert below right? It seems to be expecting a timeout indeed.

Reply:
are we sure that the intention is to time out as soon as possible, or to wait enough time? I wonder why 10 was chosen.

Reply:
It's very likely because that was short enough on the developer's machine to trigger the timeout. The problem is, on a fast enough machine or with enough improvements to the code under test, 10ms might not be long enough.

Reply:
IOW, this test is currently trying to wait 10ms of monotonic time and seeing if it can or can't do work in that time. But that's susceptible to flakes.